Formula/ccls.rb

```diff
@@ -0,0 +1,37 @@
+class Ccls < Formula
+  desc "C/C++ language server"
+  homepage "https://github.com/MaskRay/ccls"
+  url "https://github.com/MaskRay/ccls.git", :tag => "0.20180513", :revision => "19d0aad2ca9663b5633b575efe41803005b21d38"
```

Comment:
There is a more recent release. Please build from source archive (https://github.com/MaskRay/ccls/archive/0.20180521.tar.gz) rather than git tag.

Reply:
tar.gz only contains the URL of the submodules, without COMMIT ID don't know which version of code to fetch.

Reply:
OK for tag then. But use the latest please.